scripts/ContentTweaker.zs

```diff
@@ -76,17 +76,60 @@ unfinished_vacuum_tube.register();
 var engine_block_cast = VanillaFactory.createItem("engine_block_cast");
 var full_engine_block_cast = VanillaFactory.createItem("full_engine_block_cast");
 var engine_block = VanillaFactory.createItem("engine_block");
+var silicon_dioxide_wafer = VanillaFactory.createItem("silicon_dioxide_wafer");
+var photocoated_wafer = VanillaFactory.createItem("photocoated_wafer");
+var engraved_wafer = VanillaFactory.createItem("engraved_wafer");
+var etched_wafer = VanillaFactory.createItem("etched_wafer");
+var lithography_mask = VanillaFactory.createItem("lithography_mask");
+var photocoated_board = VanillaFactory.createItem("photocoated_board");
+var engraved_board = VanillaFactory.createItem("engraved_board");
```

Comment:
to be fair it's better to have this stuff in a GT addon, because every single item in contenttweaker adds 1 ID, and we have a max of 4096 ID's

Reply:
We have a max of 4096 IDs *for now*